lib/notifications/client/request_error.rb

```diff
@@ -18,5 +18,38 @@ def message_from(body)
         body
       end
     end
+
+    class ClientError < RequestError; end
+    class BadRequestError < RequestError; end
```

Comment:
I think this one and the following 3 should inherit from `ClientError`

Reply:
Yep, good catch! Fixed in 0fcbbd7db006c1f3f716c15624b5ccb671178ef5.